### PR TITLE
[Documentation] update the SD - GPU page

### PR DESCRIPTION
**Pull request by AlekseyKorzik**
## Summary by CodeRabbit

- **Documentation**
	- Enhanced the formatting and structure of the stable diffusion GPU tutorial for improved clarity and readability. This includes refined command line instructions, updated installation commands, and streamlined script execution processes.

### CODE REVIEW

File: docs/docs/examples/model-inference/stable-diffusion-gpu/index.md [hunk 1]
```diff
@@ -4,25 +4,31 @@ sidebar_position: 6
...
 
-This example tutorial demonstrates how to use stable diffusion on a GPU and run it on the [Bacalhau](https://www.bacalhau.org/) network.[Stable Diffusion](https://github.com/CompVis/stable-diffusion) is a state of the art text-to-image model that generates images from text and was developed as an open-source alternative to [DALL·E 2](https://openai.com/dall-e-2/). It is based on a [Diffusion Probabilistic Model](https://arxiv.org/abs/2102.09672) and uses a [Transformer](https://arxiv.org/abs/1706.03762) to generate images from text.
+This example tutorial demonstrates how to use stable diffusion on a GPU and run it on the [Bacalhau](https://www.bacalhau.org/) network. [Stable Diffusion](https://github.com/CompVis/stable-diffusion) is a state of the art text-to-image model that generates images from text and was developed as an open-source alternative to [DALL·E 2](https://openai.com/dall-e-2/). It is based on a [Diffusion Probabilistic Model](https://arxiv.org/abs/2102.09672) and uses a [Transformer](https://arxiv.org/abs/1706.03762) to generate images from text.
```
Comment (rossjones) [inline docs/docs/examples/model-inference/stable-diffusion-gpu/index.md:8]:
```suggestion
This example tutorial demonstrates how to use stable diffusion on a GPU and run it on the [Bacalhau](https://www.bacalhau.org/) demo network. [Stable Diffusion](https://github.com/CompVis/stable-diffusion) is a state of the art text-to-image model that generates images from text and was developed as an open-source alternative to [DALL·E 2](https://openai.com/dall-e-2/). It is based on a [Diffusion Probabilistic Model](https://arxiv.org/abs/2102.09672) and uses a [Transformer](https://arxiv.org/abs/1706.03762) to generate images from text.
```

File: docs/docs/examples/model-inference/stable-diffusion-gpu/index.md [hunk 2]
```diff
@@ -180,7 +173,7 @@ The following presents additional parameters you can try:
 
 - `python main.py --p "cat with three eyes`  - to set prompt
 
-- `python main.py --p "cat with three eyes` --n 100 - to set the number of iterations to 100
+- `python main.py --p "cat with three eyes --n 100` - to set the number of iterations to 100
```
Comment (rossjones) [inline docs/docs/examples/model-inference/stable-diffusion-gpu/index.md:176]:
```suggestion
- `python main.py --p "cat with three eyes --n 100"` - to set the number of iterations to 100
```

**Review by rossjones — APPROVED** [review]
Couple of suggestions, but good to merge after they're fixed.

**Comment by AlekseyKorzik** [comment]
Fixed

**Pull request merged**